### ISSUE TITLE
Edge attributes for BIO dataset

### ISSUE THREAD

**Issue by mdanb** [snap-stanford/pretrain-gnns#30]
In the paper, you describe the BIO dataset as having 7 edge attributes corresponding to different interactions (coexpression, neighborhood etc...). However, when I stepped through the code, I noticed that `edge_attr` for geometric_data_processed.pt has 9 features. Was wondering about the discrepancy.

**Comment by weihua916** [comment]
Hi! 

According [this line](https://github.com/snap-stanford/pretrain-gnns/blob/master/bio/loader.py#L57-L60), the 8-th and 9-th edge features are used for self-loop and mask pre-training (we set 9-th element to be 1 to indicate that the edge has been masked, see [here](https://github.com/snap-stanford/pretrain-gnns/blob/7bb81b5cc2d37241ee72cbfa40fbd89b0cc2394f/bio/util.py#L99-L101)). 

> EDIT: Additionally, I'm unsure why the Embedding for node features has dimension 2 x 300. Why 2?
I think you are referring to [this line](https://github.com/snap-stanford/pretrain-gnns/blob/7bb81b5cc2d37241ee72cbfa40fbd89b0cc2394f/bio/model.py#L32). This is because all the node features are uniformly 1 (see [this](https://github.com/snap-stanford/pretrain-gnns/blob/7bb81b5cc2d37241ee72cbfa40fbd89b0cc2394f/bio/loader.py#L47-L48)). It's true that we did not learn 0-th embedding...
